go.mod

```diff
@@ -1,9 +1,8 @@
-module github.com/workos/workos-go/v3
+module github.com/workos/workos-go/v4
 
 go 1.13
 
 require (
 	github.com/google/go-querystring v1.0.0
-	github.com/google/uuid v1.1.1
```

Comment:
what's the context for this change?

Reply:
I just ran `go mod tidy`, the dependency was previously removed in the codebase but not updated in the `go.mod`